science-china-life-sciences.csl

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
```

Comment:
@damnation333, http://life.scichina.com:8082/sciCe/EN/column/column13292.shtml says "SCIENCE CHINA Life Sciences (Sci China Life Sci) ... is published monthly in English", so `<style>` should have a "default-locale" attribute set to either "en-US" (US English) or "en-GB" (British English). (whatever works best for this style; the main differences are the date formats and quotation marks)